server/src/apiClient/SwapiClient.js

```diff
@@ -0,0 +1,28 @@
+import got from "got"
+
+class SwapiAPI {
+    static async get() {
```

Comment:
name method as `getFilms` to be specific about the information it's requesting (will help later when you want to query for other data from swapi)